app/controllers/trainer_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TrainerController < ApplicationController
+  before_action :set_card, only: [:review]
+  respond_to :html
+
+  def review
+    if @card.check_user_translation(trainer_params[:user_translation])
+      flash[:alert] = 'Вы ввели верный перевод. Продолжайте.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.